testsuite/MDAnalysisTests/analysis/test_dielectric.py

```diff
@@ -0,0 +1,75 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
```

Comment:
Should the copyright notice span the correct range of years? Not sure what our policy on this has been.

Reply:
I also don't know. But at least we could put 2022.

Reply:
We've been super inconsistent here 😕 I guess it doesn't matter so much?

We mostly need a decision on https://github.com/MDAnalysis/mdanalysis/pull/2118#issuecomment-1032863726

Reply:
Dear @IAlibay, I apologize for being unresponsive but I have not been active in the field of computational chemistry for several years. Let me know what kind of approval you need in order to proceed with the PR.

Reply:
Hi @mattiafelice-palermo !

We are currently exploring if it is desirable and feasible to change the license of MDAnalysis to something more permissive. We will soon poll the contributors, but we need a way to reach them all.

We need a way to reach you for that poll or need to know if you are OK with your contribution to MDAnalysis to be relicensed to a more permissive open source license. If you are not OK with that, it is not a blocker, we'll just add that on the things to consider.

Cheers.

Reply:
@jbarnoud Thanks for the thorough explanation! You can reach me through GitHub, or if an email address is needed, I can PM it to you. I am more than ok with relicensing the code to a more permissive license :)